net/aircrack-ng/Makefile

```diff
@@ -8,27 +8,41 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aircrack-ng
-PKG_VERSION:=1.2-rc1
-PKG_RELEASE:=3
+PKG_VERSION:=1.4
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://download.aircrack-ng.org/ \
-		http://archive.aircrack-ng.org/aircrack-ng/$(PKG_VERSION)/
-PKG_HASH:=cf3134521e1c3d7aed4e384e3e5e7b6959e2d485bd1554474608a3a9328e35fd
+PKG_SOURCE_URL:=https://github.com/aircrack-ng/aircrack-ng/archive/$(PKG_VERSION)/
```

Comment:
It's preferred to use official tarballs instead of github generated tarballs. So the original URL in https.

Otherwise with github, it's preferred to use codeload directly instead of the redirect.

Reply:
Fixed, using codeload.